packages/sanes-chrome-extension/src/routes/account/index.tsx

```diff
@@ -81,15 +81,20 @@ const AccountView = (): JSX.Element => {
         history.replace(WELCOME_ROUTE);
       },
     },
-    {
-      icon: termsIcon,
-      text: 'Terms & Conditions',
-      action: () => window.open('https://support.iov.one/hc/en-us', '_blank'),
-    },
   ];
 
+  const footer = (
+    <Block marginTop={4} textAlign="center">
+      <Block marginBottom={1}>
+        <Typography variant="subtitle2" inline>
```

Comment:
We usually uses `Typography` as child of the `Link` component, also we have `link` prop for `Typography`.

Reply:
I changed this because in the storybook, the `<a>` and `<h6>` end up with different colors.

<img width="402" alt="Bildschirmfoto 2019-07-03 um 16 18 47" src="https://user-images.githubusercontent.com/2603011/60599184-493b3c80-9dae-11e9-8c87-7ab18793cf70.png">

However, this seems to be a storybook issue only. Will update for consistency with https://github.com/iov-one/ponferrada/blob/d3da632/packages/sanes-chrome-extension/src/routes/login/components/LoginControls.tsx

Reply:
you can use `<Typography .... inline link color="primary">....</Typography>`

Reply:
This is what is done in LoginControls, where I saw the color issue above.

But I found the issue: I open the storybook in Firefox and the extension in Chrome. Firefox uses the `<a>` color for the line (default link blue), Chrome uses the `<h6>` color for the line.

One more thing understood :)